cfme/tests/v2v/test_v2v_migrations_ui.py

```diff
@@ -243,6 +245,72 @@ def test_v2v_ui_set2(appliance, v2v_providers, form_data_single_datastore, soft_
     view = navigate_to(infrastructure_mapping_collection, 'All')
     view.migration_plans_not_started_list.migrate_plan(plan_name)
 
+    # Test Archive completed migration  plan
+    wait_for(func=view.plan_in_progress, func_args=[plan_name], delay=5, num_sec=300,
+        handle_exception=True)
+    view.migr_dropdown.item_select("Completed Plans")
+    view.wait_displayed()
+    view.migration_plans_completed_list.archive_plan(plan_name)
+    view.migr_dropdown.item_select("Archived Plans")
+    view.wait_displayed()
+    soft_assert(plan_name in view.migration_plans_archived_list.read())
+
+
+@pytest.mark.parametrize('form_data_multiple_vm_obj_single_datastore', [['nfs', 'nfs',
+    [rhel7_minimal, rhel7_minimal]]], indirect=True)
+def test_v2v_ui_migration_plan_sorting(appliance, v2v_providers,
+        form_data_multiple_vm_obj_single_datastore, soft_assert):
+    infrastructure_mapping_collection = appliance.collections.v2v_mappings
+    migration_plan_collection = appliance.collections.v2v_plans
+
+    mapping = infrastructure_mapping_collection.create(
+        form_data_multiple_vm_obj_single_datastore[0])
+
+    plan1 = migration_plan_collection.create(
+        name="plan_{}".format(fauxfactory.gen_alphanumeric()), description="desc_{}"
+        .format(fauxfactory.gen_alphanumeric()), infra_map=mapping.name,
+        vm_list=[form_data_multiple_vm_obj_single_datastore[1][0]], start_migration=False)
```

Comment:
isn't `form_data_multuiple_vm_ob_single_datastore` returning a namedtuple now? Need a comment at the very least for what we're expecting these indices to be.

Reply:
fixed.